src/picongpu/include/particles/ionization/byField/BSI/BSI_Impl.hpp

```diff
@@ -167,22 +155,18 @@ namespace ionization
                 const int particleCellIdx = particle[localCellIdx_];
                 /* multi-dim coordinate of the local cell inside the super cell */
                 DataSpace<TVec::dim> localCell(DataSpaceOperations<TVec::dim>::template map<TVec > (particleCellIdx));
-                /* interpolation of E- */
+                /* interpolation of E */
                 const fieldSolver::numericalCellType::traits::FieldPosition<FieldE> fieldPosE;
                 ValueType_E eField = Field2ParticleInterpolation()
                     (cachedE.shift(localCell).toCursor(), pos, fieldPosE());
-                /*                     and B-field on the particle position */
-                const fieldSolver::numericalCellType::traits::FieldPosition<FieldB> fieldPosB;
-                ValueType_B bField = Field2ParticleInterpolation()
-                    (cachedB.shift(localCell).toCursor(), pos, fieldPosB());
 
                 /* define number of bound macro electrons before ionization */
                 float_X prevBoundElectrons = particle[boundElectrons_];
```

Comment:
I will do a small change to that but refactoring the interface would also require me to change `ionization.hpp` and `WriteElectronIntoFrame` from `ionizationMethods.hpp`. That should be a separate PR.